incubator/virtualcluster/pkg/syncer/controllers/configmap/controller.go

```diff
@@ -107,17 +105,12 @@ func (c *controller) Reconcile(request reconciler.Request) (reconciler.Result, e
 
 func (c *controller) reconcileConfigMapCreate(cluster, namespace, name string, configMap *v1.ConfigMap) error {
 	targetNamespace := conversion.ToSuperMasterNamespace(cluster, namespace)
-	newObj, err := conversion.BuildMetadata(targetNamespace, configMap)
+	newObj, err := conversion.BuildMetadata(cluster, targetNamespace, configMap)
 	if err != nil {
 		return err
 	}
 
-	innerCluster := c.multiClusterConfigMapController.GetCluster(cluster)
-	client, err := clientset.NewForConfig(restclient.AddUserAgent(innerCluster.GetClientInfo().Config, "syncer"))
-	if err != nil {
-		return err
-	}
-	_, err = client.CoreV1().ConfigMaps(targetNamespace).Create(newObj.(*v1.ConfigMap))
+	_, err = c.configMapClient.ConfigMaps(targetNamespace).Create(newObj.(*v1.ConfigMap))
```

Comment:
check already exists error here

Reply:
Maybe we should log err here since the err is dropped in line 90.